vaccine_feed_ingest/runners/il/ilvaccine_org/normalize.py

```diff
@@ -0,0 +1,119 @@
+#!/usr/bin/env python
+
+import datetime
+import json
+import os
+import pathlib
+import re
+import sys
+from typing import List, Optional
+
+from vaccine_feed_ingest_schema import location as schema
+
+from vaccine_feed_ingest.utils.log import getLogger
+
+logger = getLogger(__file__)
+
+ADDRESS_RE = re.compile(r"(.*)(?:,|\n)([a-zA-Z. ]+),\s?(?:IL|Illinois)?\s+(\d{5})")
+
+# Rewrites for addresses that don't quite fit the regex.
+ADDRESS_FIXES = {
+    "3401 N Knox Ave Chicago, IL 60641": "3401 N Knox Ave\nChicago, IL 60641",
+    "2674 N Halsted St Chicago IL 60614\n": "2674 N Halsted St\nChicago, IL 60614",
+    "701 W North Ave, Melrose Park, IL": "701 W North Ave, Melrose Park, IL 60160",
+    ".": "2751 W Winona St, Chicago, IL 60625",  # Swedish Hospital, Chicago
```

Comment:
I'm a little nervous about mapping `.` to something incase the upstream issue re-occurs. Would it maybe be better to drop this particular address?